pypit/scripts/run_pypit.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+
+"""
+This script pushes a FITS file to ginga
+"""
+from __future__ import (print_function, absolute_import, division,
+                        unicode_literals)
+
+try:
+    from xastropy.xutils import xdebug as debugger
+except:
+    import pdb as debugger
+
+def parser(options=None):
```

Comment:
Can msgs.usage() be called?  I mean, the built-in  -h  function offers very limited information (e.g. it doesn't show the available spectrographs etc.). Unless I'm mistaken?  Is it possible to get a call in to msgs.usage()?